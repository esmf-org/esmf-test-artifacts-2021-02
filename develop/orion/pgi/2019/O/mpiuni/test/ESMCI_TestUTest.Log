20210104 192856.493 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210104 192856.494 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210104 192856.494 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210104 192856.494 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210104 192856.494 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210104 192856.494 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210104 192856.494 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_40-7-gccaa48f
20210104 192856.494 INFO             PET0 ESMF library build date/time: "Jan  4 2021" "18:34:11"
20210104 192856.494 INFO             PET0 ESMF library build location : /home/hpillai/esmf_test/pgi/esmf
20210104 192856.494 INFO             PET0 ESMF_COMM                   : mpiuni
20210104 192856.494 INFO             PET0 ESMF_NETCDF                 : enabled
20210104 192856.494 INFO             PET0 ESMF_PNETCDF                : disabled
20210104 192856.494 INFO             PET0 ESMF_PIO                    : disabled
20210104 192856.494 INFO             PET0 ESMF_YAMLCPP                : enabled
20210104 192856.494 INFO             PET0 ESMF_MOAB                   : enabled
20210104 192856.494 INFO             PET0 Beginning Test, file ESMCI_TestUTest.C, line 32
20210104 192856.494 INFO             PET0 NUMBER_OF_PROCESSORS 1
20210104 192856.494 INFO             PET0 PASS Use of ESMC_Test(), true, ESMCI_TestUTest.C, line 39
20210104 192856.494 INFO             PET0 Ending Test, file ESMCI_TestUTest.C, line 43
20210104 192856.494 INFO             PET0 Finalizing ESMF
Mon Jan  4 22:23:32 CST 2021
