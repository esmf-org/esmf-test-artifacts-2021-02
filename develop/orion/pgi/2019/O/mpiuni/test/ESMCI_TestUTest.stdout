/home/hpillai/esmf_test/pgi/esmf/src/Infrastructure/stubs/mpiuni/mpirun: line 41: ./ESMCI_TestUTest: No such file or directory
Sun Dec 27 20:53:09 CST 2020
